dask/array/tests/test_array_core.py

```diff
@@ -617,6 +617,22 @@ def test_store():
     assert raises(ValueError, lambda: store([at, bt], [at, bt]))
 
 
+def test_to_hdf5():
+    try:
+        import h5py
+    except ImportError:
+        return
```

Comment:
I've had some pain with it in the past.  Sticking with return for now.  If
anyone else wants to set up machinery around unittest2 I'd be much
obliged.  I'll put this on my todo list; it's fairly far down though.
Merging once tests pass.

On Fri, May 8, 2015 at 5:51 PM, Stephan Hoyer <notifications@github.com>
wrote:

> In dask/array/tests/test_array_core.py
> <https://github.com/ContinuumIO/dask/pull/191#discussion_r29987372>:
>
> > @@ -617,6 +617,22 @@ def test_store():
> >      assert raises(ValueError, lambda: store([at, bt], [at, bt]))
> >
> >
> > +def test_to_hdf5():
> > +    try:
> > +        import h5py
> > +    except ImportError:
> > +        return
>
> It's a test only dependency available on conda and pypi. Up to you, but
> really not too bad in my opinion. It was a little bit of work up front, but
> really hasn't been much trouble at all for xray.
>
> I think there is significant value to showing skipped tests -- it makes it
> very obvious to users running the test suite if they are getting complete
> coverage or not (e.g., because they have some missing dependencies).
>
> If somebody is still running Python 2.6, they are used to annoyances like
> needing to install unittest2 before running test suites.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuumIO/dask/pull/191/files#r29987372>.
>